src/explore-education-statistics-frontend/src/components/UserTestingBanner.tsx

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+import useMounted from '@common/hooks/useMounted';
+import useToggle from '@common/hooks/useToggle';
+import { useCookies } from '@frontend/hooks/useCookies';
+import React, { useEffect } from 'react';
+import VisuallyHidden from '@common/components/VisuallyHidden';
+import ButtonText from '@common/components/ButtonText';
+import styles from './UserTestingBanner.module.scss';
+import Link from './Link';
+
+const UserTestingBanner = () => {
+  const { getCookie, setUserTestingBannerSeenCookie } = useCookies();
+  const isUserTestingBannerSeen = getCookie('userTestingBannerSeen') === 'true';
+  const [isVisible, toggleVisible] = useToggle(!isUserTestingBannerSeen);
+  const { isMounted } = useMounted();
+
+  useEffect(() => {
+    toggleVisible(!isUserTestingBannerSeen);
+  }, [isUserTestingBannerSeen, toggleVisible]);
+
+  return isMounted && isVisible ? (
+    <div className={styles.container}>
+      <div className="govuk-width-container">
+        <div className="govuk-grid-row">
+          <div className="govuk-grid-column-three-quarters govuk-!-margin-bottom-0">
+            <p className="govuk-!-margin-bottom-2">
+              <Link to="https://forms.office.com/Pages/ResponsePage.aspx?id=yXfS-grGoU2187O4s0qC-XMiKzsnr8xJoWM_DeGwIu9UQVNYVkxZSEJVVjhPOURXSjJVMjhZRTdYMi4u">
+                Get involved in making Explore education statistics better
+              </Link>
+              <VisuallyHidden> and help us improve the service</VisuallyHidden>
```

Comment:
Why this visually hidden text? 

The link text says everything we need so this doesn't seem to add anything. It's a little confusing to only say it to screen readers as well.

Reply:
Think this was just a hangover from the initial implementation